DQMOffline/Trigger/python/Tau3MuMonitor_cff.py

```diff
@@ -0,0 +1,31 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.Tau3MuMonitor_cfi import hltTau3Mumonitoring
+
+HLT_Tau3Mu_Mu5_Mu1_TkMu1_Tau10_Monitoring                                             = hltTau3Mumonitoring.clone()
+HLT_Tau3Mu_Mu5_Mu1_TkMu1_Tau10_Monitoring.FolderName                                  = cms.string('HLT/BPH/Tau3Mu/Mu5_Mu1_TkMu1_Tau10')
+HLT_Tau3Mu_Mu5_Mu1_TkMu1_Tau10_Monitoring.taus                                        = cms.InputTag('hltTauPt10MuPts511Mass1p2to2p3Iso', 'Taus') # 3-muon candidates
```

Comment:
uhmm, so are you going to have this module run on HLTMonitor PD ?
otherwise I doubt you will have the needed collection as input ....
if so, I think you need to update your JIRA ticket and request to add this collection to the output for the HLTMonitor PD and the other output used by relval, for instance